tests/functional_tests/transfer.py

```diff
@@ -57,6 +57,15 @@ def diff_incoming_transfers(actual_transfers, expected_transfers):
     # wallet2 m_transfers container is ordered and order should be the same across rescans
     diff_transfers(actual_transfers, expected_transfers, ignore_order = False)
 
+def restore_wallet(wallet, seed, restore_height = 0, filename = '', password = '', ):
```

Comment:
The trailing `, ` after the last param, though allowed, seems to break with the style elsewhere in this file, as well as python convention.

Reply:
Fixed